web/src/hooks/getEstimationTranslation.ts

```diff
@@ -0,0 +1,21 @@
+import { useTranslation } from 'react-i18next';
+
+export function useGetEstimationTranslation(
+  field: 'title' | 'pill' | 'body',
+  estimationMethod?: string,
+  estimatedPercentage?: number
+) {
+  const { t } = useTranslation();
+
+  const exactTranslation =
+    (estimatedPercentage ?? 0) > 0 && estimationMethod === 'aggregated'
+      ? t(`estimation-card.aggregated_estimated.${field}`, {
+          percentage: estimatedPercentage,
+        })
+      : t(`estimation-card.${estimationMethod?.toLowerCase()}.${field}`);
```

Comment:
Should we change the enums to be lower case by default so we don't need to call `toLowerCase()`?

Reply:
Sometimes they are taken from the api object and are uppercase or lowercase, I'll make it so the translations and enums match the casing but I think in the future could improve this by matching this as one case between front and backend.